src/main/java/uk/gov/gchq/magmacore/demo/ObjectDatabaseDemo.java

```diff
@@ -40,17 +39,13 @@ public final class ObjectDatabaseDemo {
      * Run the in-memory Object Database example.
      */
     public void run() {
-        final MagmaCoreObjectDatabase objectDatabase = new MagmaCoreObjectDatabase();
-
-        // Create set of example data objects.
-        final List<Thing> objects = ExampleDataObjects.createDataObjects();
+        final var mcService = MagmaCoreServiceFactory.createWithObjectDatabase();
 
         // Add example data objects to dataset.
-        objects.forEach(objectDatabase::create);
+        ExampleDataObjects.populateExampleData(mcService);
```

Comment:
Executing this example produces the following error: 
> An exception occured while executing the Java class. class uk.gov.gchq.hqdm.model.impl.SpatioTemporalExtentImpl cannot be cast to class uk.gov.gchq.hqdm.model.KindOfPerson (uk.gov.gchq.hqdm.model.impl.SpatioTemporalExtentImpl and uk.gov.gchq.hqdm.model.KindOfPerson are in unnamed module of loader java.net.URLClassLoader @a5e8260)

Reply:
I will add a 'regression' type unit test so this kind of error is caught in future, and I'll fix the problem.

Reply:
Thank you 😊

Reply:
Can it! 🗑️